R/zchunk_LA132.industry.R

```diff
@@ -48,48 +48,160 @@ module_energy_LA132.industry_DISABLED <- function(command, ...) {
     L131.share_R_Senduse_heat_Yh <- get_data(all_data, "L131.share_R_Senduse_heat_Yh")
 
```

Comment:
Please silence package check notes.

Reply:
Could you please elaborate a bit? Did you mean there is an option in `get_data`?

Reply:
This may be easier to show than explain, but here's Robert's May 19 Slack post on the subject:

>Good news, everyone!  You might have noticed that when you do your package check before pushing your branch up to GitHub, you now no longer see a bunch of “no visible binding” messages.  This should make it a lot easier to see real errors when they occur.  Huzzah!  We’d like to keep it this way, so henceforth each developer will be responsible for suppressing those messages in any chunks that they develop.  The messages come about because of the way that dplyr functions allow you to use bare symbols in expressions like `select(somedata, iso, GCAM_region_ID)`.  The fix is to give these symbols some value before you mention them.  It doesn’t matter what the value is, since it won’t actually be used, so just use `NULL`.  Just add an assignment right at the top of your chunk.  In this case it would look like `iso <- GCAM_region_ID <- NULL`.  Consider this the last step in your pre-push checks. Picking up your litter helps keep the place clean for everyone.

So: do a Build->Check Package, looks for "no visible binding" messages for variables in this package, and set them to NULL. See any other existing chunk for an example of this.

If this is unclear, stop by, and we can do it together. Thanks.